### ISSUE TITLE
TypeError: low(...).state is not a function.

### ISSUE THREAD

**Issue by flexzuu** [typicode/json-server#289]
Hi i get the following error since version 0.8.12 and 0.8.13 version 0.8.11 is working fine.

```
> json-server ./api/db.json --port 3004


  \{^_^}/ hi!

  Loading ./api/db.json
/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/src/cli/utils/load.js:31
    data = low(source, { storage: fileAsync }).state()
                                               ^

TypeError: low(...).state is not a function
    at module.exports (/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/src/cli/utils/load.js:31:48)
    at start (/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/src/cli/run.js:99:5)
    at module.exports (/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/src/cli/run.js:126:3)
    at module.exports (/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/src/cli/index.js:86:3)
    at Object.<anonymous> (/Users/jonas/Documents/Projects/react-redux-axios-boilerplate/node_modules/json-server/bin/index.js:2:22)
    at Module._compile (module.js:541:32)

npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/jonas/.nvm/versions/node/v6.1.0/bin/node" "/Users/jonas/.nvm/versions/node/v6.1.0/bin/npm" "run" "api"
npm ERR! node v6.1.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE
npm ERR! react-redux-axios-boilerplate@0.1.0 api: `json-server ./api/db.json --port 3004`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-redux-axios-boilerplate@0.1.0 api script 'json-server ./api/db.json --port 3004'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-redux-axios-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     json-server ./api/db.json --port 3004
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-redux-axios-boilerplate
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-redux-axios-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jonas/Documents/Projects/react-redux-axios-boilerplate/npm-debug.log
```

**Comment by kesavananandham** [comment]
i'm also  getting the same issue :disappointed: in 0.8.13

**Comment by johnlindquist** [comment]
I'm thinking this is a node v6 issue...

**Comment by cliffordfajardo** [comment]
I'm getting this same error on Node version `v5.11.0`.

**Comment by johnlindquist** [comment]
@cliffordfajardo it seems to be one of the dependencies. If you remove all of the "^" from `package.json`, then `npm install`, it works fine.

**Comment by cliffordfajardo** [comment]
@johnlindquist -  I remove all of the `^` from the version numbers of my `package.json`  & did a clean install & still no success.

**Comment by johnlindquist** [comment]
@cliffordfajardo it's `lowdb@0.13.0-beta.3` that breaks it. `beta.2` is fine.

cc @typicode

**Comment by typicode** [comment]
I've released a fix (`v0.8.14`) and I've deprecated the bad releases.
Sorry all for the inconvenience and thanks @johnlindquist for the help.

Let me know if you have other issues.
